changelog.asc

```diff
@@ -2,6 +2,41 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.20.0 (20241204)
+
+In 2020, implementing Anki's latest scheduling improvements would have taken years. Today, the same process takes weeks due to the extensive effort of merging Anki's codebase into AnkiDroid. Enjoy your even more efficient reviews!
+
+* Your https://opencollective.com/ankidroid/contribute[donations] paid for the work to make this happen so quickly 🤗
+* *AnkiDroid Updates*
+** Includes https://apps.ankiweb.net/[Anki 24.11], with https://github.com/open-spaced-repetition/fsrs4anki[FSRS 5.0]
+*** If you use FSRS, we recommend re-optimizing parameters
+** Forget Cards: Add link to manual
+** Deck Overview: Re-include 'total cards' statistics
+* *Anki 24.11 Features*
+** FSRS 5.0
+*** Haven't tried the new FSRS scheduler? You can enable it in deck options
```

Comment:
I had to look it up, thought it was "System" ? Which made me think, it's all new, let's define it
edited out everything else around it to give it room. ?

```suggestion
*** Want to try FSRS (the Free Spaced Repetition Scheduler)? Enable it in deck options
```